internal/cmd/operator-sdk/cli/cli.go

```diff
@@ -66,22 +69,26 @@ func Run() error {
 // the kubebuilder project layout
 func GetPluginsCLIAndRoot() (*cli.CLI, *cobra.Command) {
 	ansibleBundle, _ := plugin.NewBundle("ansible"+plugins.DefaultNameQualifier, plugin.Version{Number: 1},
+		commonv3.Plugin{},
 		ansiblev1.Plugin{},
 		manifestsv2.Plugin{},
 		scorecardv2.Plugin{},
 	)
-	gov2Bundle, _ := plugin.NewBundle(golangv2.Plugin{}.Name(), golangv2.Plugin{}.Version(),
+	gov2Bundle, _ := plugin.NewBundle(golang.DefaultGoNameQualifier, golangv2.Plugin{}.Version(),
+		commonv2.Plugin{},
 		golangv2.Plugin{},
 		envtestv1.Plugin{},
 		manifestsv2.Plugin{},
 		scorecardv2.Plugin{},
 	)
-	gov3Bundle, _ := plugin.NewBundle(golangv3.Plugin{}.Name(), golangv3.Plugin{}.Version(),
+	gov3Bundle, _ := plugin.NewBundle(golang.DefaultGoNameQualifier, golangv3.Plugin{}.Version(),
+		commonv3.Plugin{},
```

Comment:
We might not need to use the common here. see: https://github.com/kubernetes-sigs/kubebuilder/pull/2112

Reply:
In order to do this, we need to export the bundle, which I have planned to do once kubernetes-sigs/kubebuilder#2106 is merged. (I'm not doing it right now to prevent rebases)

Reply:
we can do that in a follow-up. I will add a comment